pkg/services/annotations/annotationsimpl/store_mock.go

```diff
@@ -0,0 +1,428 @@
+// Code generated by mockery v2.37.1. DO NOT EDIT.
```

Comment:
We do not use mock generators anymore. IIRC the backend platform recommends writing fakes\mocks\stubs manually.